src/shared/utils/urlGenerator.test.ts

```diff
@@ -0,0 +1,203 @@
+import { v4 as uuidv4 } from 'uuid';
+
+import { mockedAppletId } from 'shared/mock';
+import {
+  Path,
+  uuidRegexp,
+  getAppletPageRegexp,
+  getBuilderAppletUrl,
+  getUpdatedAppletUrl,
+  getAppletActivityPageRegexp,
+  getAppletActivityFlowPageRegexp,
+  getAppletPerformanceActivityPageRegexp,
+  APPLET_PAGE_REGEXP_STRING,
+  ACTIVITY_PAGE_REGEXP_STRING,
+  ACTIVITIES_PAGE_REGEXP_STRING,
+  ACTIVITY_FLOW_PAGE_REGEXP_STRING,
+  checkIfAppletUrlPassed,
+  checkIfAppletSettingsUrlPassed,
+  checkIfAppletActivityUrlPassed,
+  checkIfPerformanceTaskUrlPassed,
+  checkIfAppletActivityFlowUrlPassed,
+  checkCurrentAppletPage,
+  isNewApplet,
+  checkCurrentActivityPage,
+  checkCurrentActivityFlowPage,
+} from './urlGenerator';
+
+const mockedUuid = uuidv4();
+const mockedActivityOrFlowId = uuidv4();
+const mockedItemId = uuidv4();
+const mockedPath = 'path';
+const mockedPerfTaskUndefinedPath = `${ACTIVITIES_PAGE_REGEXP_STRING}\\/${Path.PerformanceTask}\\/undefined/(${uuidRegexp})`;
+const mockedPerfTaskDefinedPath = `${ACTIVITIES_PAGE_REGEXP_STRING}\\/${Path.PerformanceTask}\\/${mockedPath}/(${uuidRegexp})`;
+
+describe('get*Regexp', () => {
```

Comment:
why with "*" ? :)

Reply:
* means ['AppletPage', 'AppletActivityPage', 'AppletActivityFlowPage', 'AppletPerformanceActivityPage']
I grouped the same utilities into describes with appropriate names and replaced all changing name parts with '*', but in every test I added full util name for every case